src/phoneField/specifications/custom.md

```diff
@@ -0,0 +1,34 @@
+# Custom PhoneField
+
+```js
+import { PhoneField } from '@blablacar/ui-library'
+```
+
+## Playground
+
+<!-- STORY -->
+
+## Custom usage
+
+```js
+<div>
+  <h1 id="labelId">Accessibility label</h1>
+  <PhoneField
+    id="phoneFieldId"
+    name="phoneField"
+    className="additionalClass"
+    ariaLabelledBy="labelId"
+    selectFieldLabel="Phone country code"
+    textFieldTitle="Phone number"
+    textFieldPlaceholder="Phone number"
+    defaultRegionValue="FR"
+    defaultPhoneValue="0778674534"
+    countriesWhiteList={['FR', 'ES']}
+    customCountriesName={{
```

Comment:
nit:
countriesWhiteList => countryWhitelist
customCountriesName => customCountryNames

Reply:
I finally do a logic renaming of const and types.